garnet/src/WheelSliderControllerSample.js

```diff
@@ -45,12 +32,6 @@ var WheelSliderControllerPanel = kind({
 			this.$.sampleValue.setAttribute('aria-live', 'assertive');
 		};
 	}),
-	tapCancel: function(inSender, inEvent) {
-		this.doResult({msg: 'Cancel button tapped !!'});
-	},
-	tapOK: function(inSender, inEvent) {
-		this.doResult({msg: 'OK button tapped !!'});
-	},
 	changingEventHandler: function(inSender, inEvent) {
```

Comment:
So do you want to remove default handlers from WheelSliderController? Empty handlers are defined.